systems/data-plan/base-rate/pkg/db/rate_repo_test.go

```diff
@@ -127,7 +127,7 @@ func Test_Rates_Get(t *testing.T) {
 				"2G",
 				"3G",
 				"",
-				"INTER_MNO_DATA",
+				3,
```

Comment:
Using the type enum (instead of the number) will make it easier to update/debug in the future

Reply:
addressed